bootstrap-salt.sh

```diff
@@ -2626,7 +2626,17 @@ install_debian_check_services() {
 #
 #   Fedora Install Functions
 #
+
+FEDORA_PACKAGE_MANAGER="yum"
+
+fedora_get_package_manager() {
```

Comment:
@mbologna, would you mind renaming this to `__fedora_get_package_manager`?  Thanks.